src/Messages.js

```diff
@@ -177,20 +177,20 @@ export default defineMessages({
         description: 'System inventory - description',
         defaultMessage: 'Systems running insights-client'
     },
-    systemInventoryWarning: {
-        id: 'systemInventoryWarning',
-        description: 'System inventory - warning',
-        defaultMessage: 'stale systems'
+    systemInventoryStaleWarning: {
+        id: 'systemInventoryStaleWarning',
+        description: 'System inventory - stale warning',
+        defaultMessage: '{count} systems to be removed'
```

Comment:
likely gonna have to do the plural thing here, systems vs system s done elsewhere in this file, shouldn't take too long to fix